pyrometheus/chem_expr.py

```diff
@@ -204,20 +204,20 @@ def wilke_mixture_rule_expr(sol: ct.Solution, sp, x, mu):
         8*(1 + (w[sp]/w[j]))
     ) for j in range(sol.n_species)])
 
-    
-def species_mixture_rule_expr(sol: ct.Solution, sp, mmw, x, Dij):
+
+def species_mixture_rule_expr(sol: ct.Solution, sp, mmw, x, d_ij):
     """Generate code for species mixture rule.
        See Robert Kee "Chemically Reacting Flow" book, chapter 12.
-    
-    :returns: Expression for the species diffusion mixture rule 
+
+    :returns: Expression for the species diffusion mixture rule
```

Comment:
Please change "species diffusion" to "species mass diffusion"

Reply:
Done.